fc/mef.py

```diff
@@ -971,44 +734,41 @@ def get_transform_fxn(data_beads, peaks_mef, mef_channels,
 
     # Make output transformation function
     transform_fxn = functools.partial(fc.transform.to_mef,
-                                    sc_list = sc_all,
-                                    sc_channels = mef_channel_all)
+                                      sc_list=std_crv_all,
+                                      sc_channels=mef_channel_all)
 
     if verbose:
-        np.set_printoptions(precision = prev_precision)
+        np.set_printoptions(precision=prev_precision)
 
-    if full:
+    if full_output:
         # Clustering results
         clustering_res = {}
         clustering_res['labels'] = labels
-        # Peak finding results
-        peak_find_res = {}
-        peak_find_res['peaks_ch'] = peaks_ch_all
-        if find_peaks_method == 'smoothed_mode':
-            peak_find_res['peaks_hists'] = peaks_hists_all
-        # Peak selection results
-        peak_sel_res = {}
-        peak_sel_res['sel_peaks_ch'] = sel_peaks_ch_all
-        peak_sel_res['sel_peaks_mef'] = sel_peaks_mef_all
+        # Population stats results
+        statistic_res = {}
+        statistic_res['values'] = stats_values_all
+        # Population selection results
+        selection_res = {}
+        selection_res['channel'] = selected_channel_all
+        selection_res['mef'] = selected_mef_all
         # Fitting results
         fitting_res = {}
-        fitting_res['sc'] = sc_all
-        fitting_res['sc_beads'] = sc_beads_all
-        fitting_res['sc_params'] = sc_params_all
+        fitting_res['std_crv'] = std_crv_all
+        fitting_res['beads_model'] = beads_model_all
+        fitting_res['beads_params'] = beads_params_all
```

Comment:
This is minor, but these results dictionaries could probably be made with 1-liners:
```python
clustering_res = {'labels' : labels}
statistic_res = {'values' : stats_values_all}
```
etc.